src/test/java/com/google/api/codegen/testdata/python_main_library.baseline

```diff
@@ -1374,4 +1404,3 @@ class LibraryServiceClient(object):
             empty_pb2.Empty,
             library_pb2.GetBigBookMetadata,
             options)
-
```

Comment:
I'm assuming this lost newline is due to a change in the testing framework, and not because we are generating files that don't end in newline?